aten/src/ATen/native/cpu/Loops.h

```diff
@@ -217,6 +239,7 @@ void cpu_serial_kernel(TensorIterator& iter, func_t op) {
       });
     }
   }, {0, iter.numel()});
+  iter.cast_outputs();
```

Comment:
Unrelated change, right?

Reply:
yes, it was not added when introduced type promotion.

Reply:
Good to go with it. Just making sure.